### PR TITLE
Add to homepage cards styles

### PR DESCRIPTION
**Pull request by boyoonc**

### CODE REVIEW

File: src/sass/mmui-styles/mmui_home.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@
   .mmui-link-secondary-button {
     color: $mm-white;
     border: 1px solid $mm-white;
+    test-decoration: none;
```
Comment (kanastasakis26) [inline src/sass/mmui-styles/mmui_home.scss]: test-decoration -> text-decoration

File: src/sass/mmui-styles/mmui_home_page_cards.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.mmui-home-page-card {
```
Comment (besseym) [inline src/sass/mmui-styles/mmui_home_page_cards.scss]: So this will replace mmui-home-panel styles in mmui_home.scss?

Comment (besseym) [reply src/sass/mmui-styles/mmui_home_page_cards.scss]: We should remember to remove the panel styles in mmui_home.scss after we move Metadata Catalog to these mmui-home-page-card styles.

**Comment by boyoonc** [comment]
@besseym need you to re-review this. I've copied over bootstrap's scss for breakpoint and changed their code from sm to lg

**Pull request merged**